### DISCUSSION TITLE
Tyr_ggn in metabolicTasks_CellfieConsensus file

### DISCUSSION THREAD

**Discussion by PkiwiBird** [SysBioChalmers/Human-GEM#376] [Q&A] [answered]
Task 37 of _metabolicTasks_CellfieConsensus_ has NA[c] in 'IN', because no correspondence was found between metabolite _Tyr_ggn_ with human1 metabolite ids [see here](https://github.com/SysBioChalmers/Human-GEM/issues/280#issuecomment-903583846). 
A reaction that uses _Tyr_ggn_ in vhm, the [GGNG](https://www.vmh.life/#reaction/GGNG): Tyr_ggn[c] + 8.0 udpg[c] -> ggn[c] + 8.0 h[c] + 8.0 udp[c], is similar to reaction [MAR05395](https://metabolicatlas.org/explore/Human-GEM/gem-browser/reaction/MAR05395) in human1: 8 UDP-glucose [c] + glycogenin [c] ⇒ 8 H+ [c] + 8 UDP [c] + glycogenin G8 [c].
Note that in VHM _ggn_ seems to be [Glycogenin-8 but primed](https://www.vmh.life/#metabolite/ggn).
Therefore, does this means _Tyr_ggn_could be [_glycogenin_ in human 1](MAM01996c)?

**Comment by JonathanRob** [comment, marked as answer]
@PkiwiBird yep, I think you're right. It's a weird metabolite because it actually represents a protein, but I would agree that we can assume `Tyr_ggn` from VMH is equivalent to `glycogenin` in `Human-GEM`.
